src/script/page/preferences/accountPreferences/AvatarInput.tsx

```diff
@@ -85,30 +88,40 @@ const AvatarInput: React.FC<AvatarInputProps> = ({selfUser, isActivatedAccount,
     }
   };
 
+  const handleKeyPress = (event: React.KeyboardEvent) => {
```

Comment:
we have a `handleKeyDown` function as a wrapper in KeyboardUtils, maybe you can use it here?

Reply:
I'll check it

Reply:
done